### PR TITLE
7727 remove init and use activity context

### PR DESCRIPTION
**Pull request by dominicmarmionswrve**
https://swrvedev.jira.com/browse/SWRVE-7727

private changes:
https://github.com/Swrve/swrve-sdk/pull/791

### CODE REVIEW

**Comment by dominicmarmionswrve** [comment]
@Sergio-Mira can you take a look please?

File: SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveBase.java [hunk 1]
```diff
@@ -1093,6 +1089,7 @@ public void purchase(String item, String currency, int cost, int quantity) {
         }
     }
 
+    @Override
```
Comment (Sergio-Mira) [inline SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveBase.java]: ~ ~ Why the overrides?

**Comment by Sergio-Mira** [comment]
:+1:

File: SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveBase.java (hunk 1 above)
Comment (dominicmarmionswrve) [inline SwrveCommonSDK/src/main/java/com/swrve/sdk/SwrveBase.java]: If the interface method signature changes then this makes it far easier to refactor.

**Pull request merged**